marginalia.el

```diff
@@ -730,16 +762,47 @@ These annotations are skipped for remote paths."
       (marginalia--fields ("*Remote*" :face 'marginalia-documentation))
     (when-let (attributes (file-attributes (substitute-in-file-name (marginalia--full-candidate cand)) 'string))
       (marginalia--fields
-       ((file-attribute-modes attributes) :face 'marginalia-file-modes)
-       ((format "%s:%s"
-                (file-attribute-user-id attributes)
-                (file-attribute-group-id attributes))
+       ((concat (unless (string= user-login-name (file-attribute-user-id attributes))
```

Comment:
integer comparison here

Reply:
I just checked the docstring and it looks like this can also be an int or string
```
The UID field in ATTRIBUTES returned by file-attributes.

This is either a string or a number.  If a string value cannot be
looked up, a numeric value, either an integer or a float, is
returned.
```

Reply:
No, see my other comment. These functions are just wrappers around `nth`. What matters is the documentation of `file-attributes`.

Reply:
Changed.